src/plugin/sqlite/index.ts

```diff
@@ -0,0 +1,52 @@
+import deepmerge from 'deepmerge';
+import sqlite3 from 'sqlite3';
+import { PluginOptions } from '@/typings/config/plugin';
+import { resolvePath } from '@/utils/fs';
+
+interface Config {
+  file: string;
+  lastValue: boolean;
+  table: string;
+}
+
+const defaultConfig: Config = {
+  file: 'gimbal.db',
+  lastValue: false,
+  table: 'gimbal_archive',
+};
+
+const willNeedDatabase = (config: Config): boolean => config.lastValue;
+
+const sqlite = async ({ commandOptions, event }: PluginOptions, config: Config): Promise<void> => {
+  const sqliteConfig = deepmerge(defaultConfig, config);
+
+  if (commandOptions) {
+    sqliteConfig.file = resolvePath(commandOptions.cwd, sqliteConfig.file);
+  }
+
+  if (willNeedDatabase(sqliteConfig)) {
+    const db = new sqlite3.Database(sqliteConfig.file);
+
+    db.run(`CREATE TABLE IF NOT EXISTS ${sqliteConfig.table} (command TEXT, date INTEGER, report BLOB);`);
+
+    if (config.lastValue) {
+      const { getLastReport, saveLastReport } = await import('./last-value');
```

Comment:
Import here to save on startup times if this isn't needed, turned off by default.